api/lib/reducers/polkadotV0-reducers.js

```diff
@@ -541,7 +541,7 @@ function democracyProposalReducer(
     proposalId: proposal.index,
     networkId: network.id,
     type: proposalTypeEnum.PARAMETER_CHANGE,
-    title: `Preliminary Proposal #${proposal.index}`,
+    title: `Democracy Proposal #${proposal.index}`,
```

Comment:
Why the change?

Reply:
So the url and the title are consistent, as I understood it on Slack

Reply:
Preliminary is actually a good simplification for the user here. Can we solve this a different way?